### ISSUE TITLE
Add the ability to selectively delete local copies of a file from the cache

### ISSUE THREAD

**Issue by gscaparrotti** [jstaf/onedriver#312]
It would be very helpful to add the aforementioned feature.

This would allow the deletion of very large file without having the need to completely delete the `onedriver.db` file.

**Comment by pdvrieze** [comment]
In a similar request, it would be good to be able to manage the cache size.

**Comment by jstaf** [comment]
Yeah this is coming, currently working on doing it in https://github.com/jstaf/onedriver/issues/291

**Comment by zilpe** [comment]
Hey @jstaf, how would we go about doing this now that #291 is closed?
